app/controllers/application_controller.rb

```diff
@@ -1,5 +1,11 @@
 class ApplicationController < ActionController::Base
```

Comment:
- [ ] When running the server i am getting the following error 👇  please fix it 

![Screen Shot 2022-05-26 at 23 29 44](https://user-images.githubusercontent.com/75126481/170583542-9219606a-023a-4e4d-b408-91f0ed4626e4.png)